Unix/http/httpauth.c

```diff
@@ -793,6 +801,8 @@ static gss_buffer_t _getPrincipalName(gss_ctx_id_t pContext)
 #ifdef DEBUG
         fprintf(stderr, "srcName == NULL\n");
 #endif
+        _report_error(maj_status, min_status, "gss_display_name");
```

Comment:
If srcName is Null, then inquire context failed to get it, not gss_display_name,